CSSTooltipPseudo/explainer.md

```diff
@@ -0,0 +1,896 @@
+  - [Scenario 5: Providing custom positioning to a tooltip](#scenario-5-providing-custom-positioning-to-a-tooltip)
+- [Accessibility Considerations](#accessibility-considerations)
+  - [What is the current accessibility experience of tooltips across browsers?](#what-is-the-current-accessibility-experience-of-tooltips-across-browsers)
+  - [Proposed improvements](#proposed-improvements)
+  - [Zoom](#zoom)
+- [Security Considerations](#security-considerations)
+- [Privacy Considerations](#privacy-considerations)
+- [Open questions](#open-questions)
+- [Future ideas](#future-ideas)
+  - [Tootlip pointer/arrow](#tootlip-pointerarrow)
+  - [Customizing `::tooltip` user interactions](#customizing-tooltip-user-interactions)
+  - [CSS Anchor Positioning Tooltip Defaults](#css-anchor-positioning-tooltip-defaults)
+- [Considered alternatives](#considered-alternatives)
+  - [Alternative 1: The `interesttarget` attribute](#alternative-1-the-interesttarget-attribute)
+- [References & acknowledgements](#references--acknowledgements)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction
+
+Tooltips are used by authors to provide additional contextul information to users
+for a given interactive element on the screen. Tooltips are often presented as
+a textbox that appears on hover (or potentially some other user interaction, like
+keyboard focus, or long-press on a touchscreen device) on the element the
+tooltip is associated with.
+
+Authors can utilize the browser's built-in tooltips by setting the
+[`title`](https://html.spec.whatwg.org/multipage/dom.html#attr-title)
+attribute on an element of interest. However, the tooltip that appears as a
+result is not customizable in any way by the author.
+
+This lack of customizability has been a source of common frustration for authors
+who want to keep a consistent look and feel across all of their UI.
+
+This document explores a new `::tooltip` pseudo element that aims to provide authors
+with a low effort mechanism to adjust styles for browser built-in tooltips without
+having to create a fully custom solution, leaving positioning, input handling, and
+accessibility for their tooltips up to the browser.
+
+## Goals
+
+* Provide low effort way for authors to style core properties on browser built-in
+tooltips (for example, color properties, font properties, etc.)
+* This mechanism should handle accessibility, positioning, and input handling for
+the author by default.
+* Allow authors to override the default positioning behavior of the built-in
+tooltips.
+* Allow authors to change the timing of built-in tooltips.
+* Improve accessibility of the `title` attribute, particularly for keyboard users.
+* Guarentee security best practices, given that tooltips today can extend past
+the bounds of the browser Window.
+* Styled tooltips should produce consistent results across all platforms, ensuring
+an interoperable experience for authors.
```

Comment:
Might be worth noting two things here: 1) this is not possible today, and 2) this proposal will also allow for interoperable results between browsers on the same OS, but maybe "platforms" is generic enough to cover OS and browser.

Reply:
Since there is no way to style the default tooltips today, I don't know how best to note the first one. But I've updated the wording to add more clarity that this is specifically for styling of built-in tooltips, and also clarified interop across browsers on the same OS (although technically the goal is across OS, as well, for the styled case)